.github/templates/CITATION.cff.j2

```diff
@@ -0,0 +1,14 @@
+cff-version: 1.2.0
+type: software
+message: >
+  If you cite this software, please use these metadata.
+title: delb
+authors:
+  - family-names: Sachsenheim
+    given-names: Frank
+    email: funkyfuture@riseup.net
```

Comment:
```suggestion
  - name: The delb project posse
```

Reply:
they have two keywords for the same thing.

Reply:
i know, but given/family name lets us make explicit its a person

Reply:
i meant that there are the keywords `website` and `url` for the same thing in deifferent scopes.